src/cpu_shader/fine.rs

```diff
@@ -1,9 +1,9 @@
 // Copyright 2023 The Vello authors
```

Comment:
Much of the PTCL is still not supported and several test scenes will trigger an assertion when `--use-cpu` is enabled. Maybe we should set `let force_gpu_from = Some("fine");` for now until full support is there?

Reply:
Done.